exercises/minesweeper/canonical-data.json

```diff
@@ -20,81 +20,111 @@
       "description": "no columns",
       "property": "annotate",
       "input": {
-        "minefield": [""]
+        "minefield": [
+          "++",
+          "++"
+        ]
       },
-      "expected": [""]
+      "expected": [
+        "++",
+        "++"
+      ]
     },
     {
       "description": "no mines",
       "property": "annotate",
       "input": {
         "minefield": [
-          "   ",
-          "   ",
-          "   "
+          "+---+",
+          "|   |",
+          "|   |",
+          "|   |",
+          "+---+"
         ]
       },
       "expected": [
-        "   ",
-        "   ",
-        "   "
+        "+---+",
+        "|   |",
+        "|   |",
+        "|   |",
+        "+---+"
       ]
     },
     {
       "description": "minefield with only mines",
       "property": "annotate",
       "input": {
         "minefield": [
-          "***",
-          "***",
-          "***"
+          "+---+",
+          "|***|",
+          "|***|",
+          "|***|",
+          "+---+"
         ]
       },
       "expected": [
-        "***",
-        "***",
-        "***"
+        "+---+",
+        "|***|",
+        "|***|",
+        "|***|",
+        "+---+"
       ]
     },
     {
       "description": "mine surrounded by spaces",
       "property": "annotate",
       "input": {
         "minefield": [
-          "   ",
-          " * ",
-          "   "
+          "+---+",
+          "|   |",
+          "| * |",
+          "|   |",
+          "+---+"
         ]
       },
       "expected": [
-        "111",
-        "1*1",
-        "111"
+        "+---+",
+        "|111|",
+        "|1*1|",
+        "|1*1|",
```

Comment:
This change makes the test case no longer a mine surrounded by spaces, since now there are two mines.

Reply:
Updated.